src/main/java/genji/Parser.java

```diff
@@ -0,0 +1,120 @@
+package genji;
+
+import genji.command.Command;
```

Comment:
It is good that the imports follow a specific standard determined by you for this and the other files as well 👍